### DISCUSSION TITLE
How to add keywords when search in the overview?

### DISCUSSION THREAD

**Discussion by Srivaths-P** [end-4/dots-hyprland#303] [Q&A]
When searching in the overview (after pressing the Super key for example), I want to be able to see results based on custom keywords. For example, if I typed 'cmd', then foot should come up. Is it possible to do this?

![image](https://github.com/end-4/dots-hyprland/assets/41336229/8f2fbff6-2e6a-4d8a-91d1-4df965e47b07)

![image](https://github.com/end-4/dots-hyprland/assets/41336229/03979da2-427f-454e-adab-cbedf5829b16)

Thank you :D

**Comment by clsty** [comment]
Copy the desktop file from global directory to user-specific directory:
```bash
cp /usr/share/applications/org.codeberg.dnkl.foot.desktop ~/.local/share/applications/
```
Then edit this copy, modify `Name=Foot` to `Name=Foot (cmd)`.

Restart AGS and you will be able to search the `Foot (cmd)` entry by typing `cmd`.

See https://wiki.archlinux.org/title/Desktop_entries for more.
